src/core/public/index.ts

```diff
@@ -38,3 +38,15 @@ export interface CoreStart {
   uiSettings: UiSettingsClient;
   chrome: ChromeStart;
 }
+
+export interface Chrome {
+  Brand: Brand;
+  Breadcrumb: Breadcrumb;
+  HelpExtension: HelpExtension;
+}
+
+export interface Notifications {
```

Comment:
as we ban using namespaces I used interfaces instead. Shouldn't we just re-export types?
```js
export {
  Brand
  Breadcrumb,
  HelpExtension
  Toast, 
  ToastInput,
  ToastsStart
}
```
That not ideal as well - with more services coming it will look like a dump + possible name collision.

Reply:
I'm not sure why we have the namespace linter rule, but I think namespaces is a better solution here. As far as I can tell, the way we export `Public` and `Server` from `core/index.ts` effectively creates a namespace anyway.

Reply:
Agree, added named exports